tera-templates/crate/details.html

```diff
@@ -0,0 +1,303 @@
+                        {%- if details.documentation_url -%}
+                            <li class="pure-menu-item">
+                                <a href="{{ details.documentation_url }}" title="Canonical documentation" class="pure-menu-link">
+                                    <i class="fa fa-fw fa-file-text"></i> Documentation
+                                </a>
+                            </li>
+                        {%- endif -%}
+
+                        {# If the release has a repository, show it #}
+                        {%- if details.repository_url -%}
+                            <li class="pure-menu-item">
+                                <a href="{{ details.repository_url }}" class="pure-menu-link">
+                                    {# If the repo link is for github, show some github stats #}
+                                    {%- if details.github -%}
+                                        <i class="fa fa-github fa-fw"></i>
+                                        <i class="fa fa-star-o fa-fw"></i> {{ details.github_stars }}
+                                        <i class="fa fa-code-fork fa-fw"></i> {{ details.github_forks }}
+                                        <i class="fa fa-exclamation-circle fa-fw"></i> {{ details.github_issues }}
+
+                                    {# If the repo link is unknown, just show a normal link #}
+                                    {%- else -%}
+                                        <i class="fa fa-code-fork fa-fw"></i> Repository
+                                    {%- endif -%}
+                                </a>
+                            </li>
+                        {%- endif -%}
+
+                        {# Show a link to the crate's Crates.io page #}
+                        <li class="pure-menu-item">
+                            <a href="https://crates.io/crates/{{ details.name }}" class="pure-menu-link"
+                                title="See {{ details.name }} in crates.io">
+                                <i class="fa fa-cube fa-fw"></i> Crates.io
+                            </a>
+                        </li>
+
+                        <li class="pure-menu-heading">Dependencies</li>
+                        <li class="pure-menu-item">
+                            <div class="pure-menu pure-menu-scrollable sub-menu">
+                                <ul class="pure-menu-list">
+                                    {# List all dependencies that the current release has #}
+                                    {%- for dep in details.dependencies -%}
+                                        <li class="pure-menu-item">
+                                            <a href="/crate/{{ dep[0] }}/{{ dep[1] }}" class="pure-menu-link">
+                                                {{ dep[0] }} {{ dep[1] }}
+                                                <i class="dependencies {{ dep[2] }}">{{ dep[2] }}</i>
```

Comment:
I assume this was already present, but why do we add the dependency as a class? That seems odd.

Reply:
I sanity-checked myself and it [was present](https://github.com/rust-lang/docs.rs/blob/master/templates/crate_details.hbs#L34), but I have no idea what it does or is or means